config/environments/production.rb

```diff
@@ -46,7 +46,12 @@
   # config.action_cable.allowed_request_origins = [ 'http://example.com', /http:\/\/example.*/ ]
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  # Local changes
+  config.force_ssl = true
+
+  # Handle STS here instead of Apache, or Rails duplicates header contents
+  # Also unset cache-control header in HTTPS vhost for same reason
+  config.ssl_options = { hsts: { preload: true } }
 
   # Include generic and useful information about system operation, but avoid logging too much
   # information to avoid inadvertent exposure of personally identifiable information (PII).
```

Comment:
Below this, please edit to be:

```ruby
# Local change to reduce log bloat
config.log_level = :warn
```

Reply:
After these edits are in place, this app's production env config can be the model for how we setup all the others. I'll make an issue that points to it before I merge this PR.

Reply:
done

Reply:
Thanks!